### PR TITLE
[docs] Add status header for logzio exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for logzio exporter

### CODE REVIEW

**Comment by mx-psi** [comment]
cc @jkowall @Doron-Bargo @yotamloe to review

**Comment by jkowall** [comment]
Not beta on our side. I can update it if you would like.

**Comment by mx-psi** [comment]
> Not beta on our side. I can update it if you would like.

@jkowall So... stable then? Feel free to suggest the correct status and @Frapschen will update it

**Comment by jkowall** [comment]
> > Not beta on our side. I can update it if you would like.
> 
> @jkowall So... stable then? Feel free to suggest the correct status and @Frapschen will update it

Yes. Stable, please. 
We also support Logs and Metrics via other exporters which are part of the collector in addition to supporting OTLP.

Thank you.

**Review by mx-psi — APPROVED** [review]
Approving by proxy

File: exporter/logzioexporter/README.md [hunk 1]
```diff
@@ -77,3 +83,6 @@ service:
     logs:
       level: debug #activate debug mode
 ```
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector#stable
```
Comment (codeboten) [inline exporter/logzioexporter/README.md:87]:
```suggestion
[stable]:https://github.com/open-telemetry/opentelemetry-collector#stable
```

**Review by codeboten — APPROVED** [review]

**Pull request merged**